apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,243 @@
+        >
+          <GridItem w="100%" colSpan={1} bg="grey.50" overflowY="scroll">
+            <VStack p="2rem" gap="2rem" h="fit-content">
+              {/* Picking Layouts */}
+              {LAYOUT_DATA.map((e) => {
+                return (
+                  <VStack
+                    key={e.layoutTypename}
+                    p="0"
+                    gap="0"
+                    cursor="pointer"
+                    onClick={() => {
+                      setSelectedLayout(e)
+                    }}
+                    _hover={
+                      selectedLayout.layoutTypename === e.layoutTypename
+                        ? {}
+                        : {
+                            '.layout-title': { color: 'base.content.brand' },
+                            '.hover-text': {
+                              opacity: 1,
+                            },
+                            '.hover-border': {
+                              borderColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.hover',
+                              ),
+                            },
+                          }
+                    }
+                    sx={
+                      selectedLayout.layoutTypename === e.layoutTypename
+                        ? {}
+                        : {
+                            '.layout-title': {
+                              transition: 'color 300ms ease-out',
+                            },
+                            '.hover-text': {
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            },
+                          }
+                    }
+                  >
+                    <Box
+                      className="hover-border"
+                      borderRadius="8px"
+                      border="2px"
+                      p="1.5rem 1.5rem 0 1.5rem"
+                      mb="1.25rem"
+                      position="relative"
+                      borderColor={
+                        selectedLayout.layoutTypename === e.layoutTypename
+                          ? 'base.divider.brand'
+                          : 'base.divider.medium'
+                      }
+                    >
+                      <Image
+                        src={e.imageSource}
+                        borderRadius="4px 4px 0px 0px"
+                        boxShadow="0px 0px 20px 0px rgba(104, 104, 104, 0.30)"
+                        alt={`Image of ${e.layoutDisplayName} layout`}
+                      />
+
+                      {selectedLayout.layoutTypename !== e.layoutTypename && (
+                        <Box
+                          className="hover-text"
+                          position="absolute"
+                          top="0"
+                          left="0"
+                          right="0"
+                          bottom="0"
```

Comment:
this is abit problematic in relation to the `position: absolute` being specified above.

what absolute does is to specify co-ordinates w regards to the nearest positioned ancestor (some element with `position` specified). in here, we're saying it should be at the top (and yet at the bottom). this is problematic because of that - should we update the problematic lines?

Reply:
You're right on this, misunderstood how the position anchor works, keeping just top and left